### PR TITLE
Changes for new release

### PR DESCRIPTION
**Pull request by agnessnowplow**
For the new fractribution release.

### CODE REVIEW

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/fractribution/index.md [hunk 1]
```diff
@@ -38,6 +39,7 @@ All variables in Snowplow packages start with `snowplow__` but we have removed t
 | Variable Name                    | Description                                                                                                                                                                          | Default   |
 | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------- |
 | `channels_to_exclude`            | List of channels to exclude from analysis (empty to keep all channels). For example, users may want to exclude the 'Direct' channel from the analysis.                               | `[]`      |
+| `channels_to_include`            | List of channels to include in the analysis (empty to keep all channels). For example, users may want to include the 'Direct' channel only in the analysis.                               | `[]`      |
```
Comment (rlh1994) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/fractribution/index.md:42]: Add to the json thingy please https://www.notion.so/keep-in-the-snow/Documentation-Updates-9d1822916caa414d9f1bb931a462f1f9?pvs=4

File: src/components/JsonSchemaValidator/dbtFractribution.js [hunk 2]
```diff
@@ -19,15 +19,20 @@ export const dbtSnowplowFractributionConfigSchema = {
...
       format: 'date',
-      title: 'Conversions Window End Date',
+      title: 'Conversion Window End Date',
       description: 'End date (UTC) of your conversion window',
     },
+    snowplow__conversion_window_days: {
+      type: 'integer',
+      title: 'Conversion Window Days',
+      description: 'Optional variable in place of conversion_window_start_date and end_date to automatically define the conversion period.',
+    },
```
Comment (rlh1994) [inline src/components/JsonSchemaValidator/dbtFractribution.js:35]: Just to be a pain, should we set minimum to 1?

**Review by rlh1994 — CHANGES_REQUESTED** [review]
Also need to add the new ones to the lists at the bottom of the config page :D

**Review by rlh1994 — APPROVED** [review]
It looks like some rebase re-dos but also it's nothing wrong, send it!

**Comment by agnessnowplow** [comment]
Not the rebase but added the fix here as I already merged that one and was lazy to open a new just for that.

**Pull request merged**